src/Makefile

```diff
@@ -135,6 +135,9 @@ install: libssu.so ssu faithpd
 	mkdir -p ${PREFIX}/include/unifrac
 	cp task_parameters.hpp ${PREFIX}/include/unifrac/
 	cp api.hpp ${PREFIX}/include/unifrac/
+	cp biom.hpp ${PREFIX}/include/unifrac/
+	cp tree.hpp ${PREFIX}/include/unifrac/
+	cp biom_interface.hpp ${PREFIX}/include/unifrac/
```

Comment:
Who needs biom.hpp, tree.hpp and biom_interface.hpp, if they are were not used in api.hpp?

Reply:
Good point, no longer needed